install/index.md

```diff
@@ -304,10 +318,26 @@ Windows users can now tap into GPU accelerated data science on their local machi
 
 <i class="fas fa-info-circle text-white"></i> When installing with Docker Desktop, if the container pull command is successful, but the run command hangs indefinitely, [ensure you're on Docker Desktop >= 4.18](https://docs.docker.com/desktop/release-notes/){: target="_blank"}.
 
+<br/>
+<div id="wsl2-sdkm"></div>
+
+### **WSL2 SDK Manager Install (Preferred Method)**
+[NVIDIA's SDK Manager](https://developer.nvidia.com/sdk-manager){: target="_blank"} gives a Windows user a Graphical User Interface (GUI) option to install RAPIDS.  It also attempts to fix any environment issues before installing RAPIDS or updating RAPIDS without any Linux knowledge.
+1. In Windows, install or update WSL2 [using Microsoft's instructions](https://docs.microsoft.com/en-us/windows/wsl/install){: target="_blank"}.
+2. Install the [latest NVIDIA Drivers](https://www.nvidia.com/download/index.aspx){: target="_blank"} on the Windows host.
+3. Download [SDK Manager's Ubuntu version from their website](https://developer.nvidia.com/sdk-manager){: target="_blank"} (requires sign up or login to NVIDIA's Developer community).  Do not install yet.  It is assumed that your home directory's `Downloads` folder is where the `.deb` file will be stored.  If not, please move `sdkmanager_[version]-[build#]_amd64.deb` file to your current Download folder.
+4. Open PowerShell command line or Windows terminal, run `wsl --install -d Ubuntu-22.04`.  This will install and start Ubuntu in your Windows host system using WSL2.  Make your **sudo** password memorable as you will need it in the next two steps.
+5. Install and run SDK Manager using  inside Ubuntu by pasting this into your command line.  You will have to enter the sudo password you created when you installed Ubuntu and also change folder location to match your home directory in windows (if you don't know it, open powershell, type `$HOME`, and hit enter)
+```sudo apt update && sudo apt install wslu -y
+sudo apt install /mnt/c/Users/[YOUR HOME DIRECTORY ON WINDOWS]/Downloads/sdkmanager_[version]-[build#]_amd64.deb
```

Comment:
We should have the users fetch the `.deb` file with `wget` instead of navigating the mounted Windows drive. This adds unnecessary complexity.

Can we provide a one or two line install that is easily copy-pasted? For example, miniforge has:
```
curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
bash Miniforge3-$(uname)-$(uname -m).sh
```

Reply:
they aren't able to fetch the `.deb` file, as it requires authentication.  I'd have also rathered that :).  Also, one of the goals of this complete solution is to use the least amount of CLI and the most of the GUI for our Windows users.